spring-boot-starter/starter-client/spring/src/main/java/org/camunda/bpm/client/spring/impl/client/ClientFactory.java

```diff
@@ -101,6 +106,51 @@ protected void addClientRequestInterceptors(ExternalTaskClientBuilder taskClient
     requestInterceptors.forEach(taskClientBuilder::addInterceptor);
   }
 
+  protected void tryConfigureCreateTimeOrder(ExternalTaskClientBuilder builder) {
+    checkForCreateTimeMisconfiguration();
+
+    if (isUseCreateTimeEnabled()) {
+      builder.orderByCreateTime().desc();
+      return;
+    }
+
+    if (isOrderByCreateTimeEnabled()) {
+      handleOrderByCreateTimeConfig(builder);
+    }
+  }
+
+  protected void handleOrderByCreateTimeConfig(ExternalTaskClientBuilder builder) {
+    String orderByCreateTime = clientConfiguration.getOrderByCreateTime();
+
+    if (STRING_ORDER_BY_ASC_VALUE.equals(orderByCreateTime)) {
+      builder.orderByCreateTime().asc();
+      return;
+    }
+
+    if (STRING_ORDER_BY_DESC_VALUE.equals(orderByCreateTime)) {
+      builder.orderByCreateTime().desc();
+      return;
+    }
+
+    throw new SpringExternalTaskClientException("Invalid value " + clientConfiguration.getOrderByCreateTime()
+        + ". Please use either \"asc\" or \"desc\" value for configuring \"orderByCreateTime\" on the client");
+  }
+
+  protected boolean isOrderByCreateTimeEnabled() {
+    return clientConfiguration.getOrderByCreateTime() != null;
+  }
+
+  protected boolean isUseCreateTimeEnabled() {
+    return clientConfiguration.getUseCreateTime() != null && clientConfiguration.getUseCreateTime();
```

Comment:
🙃 We can avoid the null check if we write it like this.
```suggestion
    return Boolean.TRUE.equals(clientConfiguration.getUseCreateTime());
```

Reply:
Done.